presto-main/src/test/java/com/facebook/presto/block/TestArrayBlock.java

```diff
@@ -134,6 +141,34 @@ public void testLazyBlockBuilderInitialization()
         assertEquals(blockBuilder.getRetainedSizeInBytes(), emptyBlockBuilder.getRetainedSizeInBytes());
     }
 
+    @Test
+    public void testEstimatedDataSizeForStats()
```

Comment:
Make sure null values are tested as you did in `assertEstimatedDataSizeForStats`. Ditto for `TestMapBlock` and `TestRowBlock`

Reply:
I apply `alternatingNullValues` that sets some values in array/map/row to null. Than in `getExpectedEstimatedDataSize` i return expected size equal to 0 if the value is null.